server/src/main/java/com/dut/healthme/dto/request/AddItemRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.dut.healthme.dto.request;
+
+import com.dut.healthme.annotation.json.JsonSnakeCaseNaming;
+import com.dut.healthme.annotation.validation.ValidItemType;
+import com.dut.healthme.entity.enums.ItemType;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonSnakeCaseNaming
+public class AddItemRequest {
+    @NotNull
+    private String name;
+    @NotNull
+    private List<String> ingredients;
+    private String description;
```

Comment:
Trong thiết kế, trường `description` là NOT NULL nên ở đây nên thêm điều kiện để validation dữ liệu trước khi lưu vào database nha
```suggestion
    @NotNull(message = "Mô tả món ăn không được để trống")
    private String description;
```